src/compiler/program.ts

```diff
@@ -14,10 +14,10 @@ namespace ts {
     
     export const version = "1.7.0";
 
-    export function findConfigFile(searchPath: string): string {
+    export function findConfigFile(searchPath: string, moduleResolutionHost: ModuleResolutionHost): string {

```

Comment:
similar to `readConfigFile` we should just pass in fileExists method instead of the host.